package.json

```diff
@@ -56,5 +56,9 @@
     "ignore": [
       "tap"
     ]
+  },
+  "optionalDependencies": {
+    "compile-json-stringify": "^0.1.2",
+    "json-strify": "^0.1.7"
```

Comment:
These should be `devDependencies`, not `optionalDependencies`.

Reply:
Now looks OK?